src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala

```diff
@@ -1,7 +1,7 @@
 package com.actian.KafkaSparkVector
 
 /*
- Copyright 2015 Actian Corporation
+ Copyright 2019 Actian Corporation
```

Comment:
not a show-stopper, recommend updating this to a range, e.g. 2015-2019

Reply:
Ok, I'll change that